src/TailoringWindow.cpp

```diff
@@ -333,8 +333,8 @@ void TailoringWindow::synchronizeTreeItemSelections(QTreeWidgetItem* treeItem)
                 }
                 break;
             }
-            //fall through
         }
+            //fallthrough
```

Comment:
This was necessary to appease the `gcc` gods. Any alternative suggestions would be appreciated.